src/pkg/build/assemblers/assembler_sif.go

```diff
@@ -77,6 +79,14 @@ func createSIF(path string, definition []byte, squashfile string) (err error) {
 		return fmt.Errorf("while creating container: %s", err)
 	}
 
+	// chown the sif file to the calling user
+	if syscall.Getuid() == 0 {
+		uID, _ := strconv.Atoi(os.Getenv("SUDO_UID"))
```

Comment:
We shouldn't be ignoring errors here, *especially* since this is touching security

Reply:
Also, I think `uid` looks better than `uID`

Reply:
ummm lint on my IDE, recommend id always should be ID.
but I agree with you